Final Project/Assets/RenderPipeline/Config/ChaengRenderPIpelineConfig.asset

```diff
@@ -13,7 +13,11 @@ MonoBehaviour:
   m_Name: ChaengRenderPIpelineConfig
   m_EditorClassIdentifier: 
   skybox: {fileID: 2800000, guid: 1153dc89e66880c4eb7adc16ba588855, type: 3}
+<<<<<<< Updated upstream
   upperAmbitent: {r: 0, g: 0, b: 0, a: 1}
   lowerAmbitent: {r: 0, g: 0, b: 0, a: 1}
+=======
+  ambitent: {r: 0, g: 0, b: 0, a: 1}
+>>>>>>> Stashed changes
   secondaryRayGeneration: 0
```

Comment:
Need to cleanup this file.  There are git conflict markers in here that will make it not compile.